File: src/quine.erl
#!/usr/bin/env escript
%% -*- erlang -*-

quine(Source) -> 
    io:format(Source ++ [$(,34] ++ Source ++ [34,$),$.,$\n]),
    halt().
    
main(_) ->
    quine("-module(quine).
-export([start/0]).

quine(Source) -> 
    io:format(Source ++ [$(,34] ++ Source ++ [34,$),$.,$\\n]),
    halt().
    
make_quine() ->
    quine").

File: src/quine.pl
#!/usr/bin/perl

$quine='#!/usr/bin/perl%c%c$quine=%c%s%c;%cprintf($quine, 10, 10, 39, $quine, 39, 10, 10);%c';
printf($quine, 10, 10, 39, $quine, 39, 10, 10);
